src/interpreter/constructors.rs

```diff
@@ -31,7 +31,7 @@ impl<S> Interpreter<S> {
             #[cfg(feature = "profile-any")]
             profiler: Profiler::default(),
             unused_balance_index: Default::default(),
-            consensus_parameters: params,
+            params: params,
```

Comment:
[nit] We _should_ be able to avoid duplicating the assignment vars if they are the same:

```suggestion
            params,
```

Reply:
true, forgot that